fontique/src/attributes.rs

```diff
@@ -191,6 +191,28 @@ impl FontStretch {
     }
 }
 
+impl FontStretch {
+    /// Creates a new stretch attribute with the given value from Fontconfig.
+    ///
+    /// The values are determined based on the [fonts.conf documentation].
+    ///
+    /// [fonts.conf documentation]: https://www.freedesktop.org/software/fontconfig/fontconfig-user.html
+    pub fn from_fontconfig(width: i32) -> Self {
+        match width {
+            50 => Self::ULTRA_CONDENSED,
+            63 => Self::EXTRA_CONDENSED,
+            75 => Self::CONDENSED,
+            87 => Self::SEMI_CONDENSED,
+            100 => Self::NORMAL,
+            113 => Self::SEMI_EXPANDED,
+            125 => Self::EXPANDED,
+            150 => Self::EXTRA_EXPANDED,
+            200 => Self::ULTRA_EXPANDED,
+            _ => Self::from_ratio(width as f32 / 100.0),
```

Comment:
If the constant values are different to this value, does that mean that there are discontinuities as you go up the stack? Is it just "hinting" to the more precise value when available?

Reply:
I think just to ensure precision

We should probably add a test (not now) that the formatting of the conversions comes out correctly.